vendor/ansible_collections/nvidia/bare_metal/plugins/modules/tenant_account_info.py

```
@@ -76,14 +80,15 @@
 account_id=dict(type='str'),
 id=dict(type='str'),
 infrastructure_provider_id=dict(type='str'),
+query=dict(type='str'),
```

Flake8 (7.3.0)

[error] continuation line missing indentation or outdented

(E122)